README

```diff
@@ -1,13 +1,78 @@
-This README is for userspace RDMA fabric library.
-
 Version @VERSION@
 Released on @DATE@
 
+Introduction
+============
+The Open Fabrics Interfaces (OFI) is a framework focused on exporting
```

Comment:
This phrasing has always seemed odd to me -- "The Open Fabrics Interfaces" sounds like it is plural.  But then it says "is..." which is a singular verb.

Can we think of better phrasing?